source/vendor/detectElementResize.js

```diff
@@ -221,7 +223,14 @@ export default function createDetectElementResize(nonce, hostWindow) {
         1,
       );
       if (!element.__resizeListeners__.length) {
-        element.removeEventListener('scroll', scrollListener, true);
+        if (element.__scrollListener__) {
```

Comment:
I think you don't need to do this on this case because `scrollListener` is declared in this function and captured here.

Reply:
The problem in this case is when you have two or more `WindowScroller`s. What happens is the first `WindowScroller` instantiates a `detectElementResize` closure and registers its own `scrollListener` to the element.

Then when we unmount if the first `WindowScroller` isn't the first one to unmount, then the `removeEventListener` call fails because the references are wrong and as a result it doesn't remove the event listener which currently leaves it dangling

Reply:
Just tested the code without it. I had a dangling event listener at some point but it might have just been a defect caused by something else so I'll remove this